build/transpile.js

```diff
@@ -1077,11 +1077,16 @@ class Transpiler {
 
         if (classNames.length > 1) {
 
+            const errors = require ('../js/base/errors')
```

Comment:
This does not belong to exchange files.

Reply:
I've updated the code now so that it is cleaner for further maintenance

Reply:
@frosty00 cool! thx!